### ISSUE TITLE
Profile: Veterans' Preference question may be inaccurate

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/team-titan#1074]
![image 1](https://user-images.githubusercontent.com/352987/44410031-8cdbdd80-a531-11e8-8741-93931fb1a693.png)

We got some feedback that an applicant was confused by the difference between our lanaguage around the VEOA eligibility question and USA Staffing's language. 

https://usajobs-team.slack.com/archives/C792H9C9Z/p1534358129000100

After reviewing the FedsHireVets.gov page on Vet's preference (https://www.fedshirevets.gov/job-seekers/special-hiring-authorities/#veoa) they clearly do include an OR statement:

> To be eligible to be considered pursuant to VEOA appointment, your latest, discharge must be issued under honorable conditions (this means an honorable or general discharge), AND you must be either:
> 
> a preference eligible (defined in title 5 U.S.C. 2108(3)), OR
> a veteran who substantially completed 3 or more years of active service under honorable conditions.

**Comment by Piglady64** [comment]
Sent a message to Kim Holden in Policy for guidance. Will post her comments as soon as I get them.

**Comment by Piglady64** [comment]
From Kim:
Julie – Thanks for reaching out.  I will have the policy team take a look at the language.  The language should be consistent on all OPM systems.  We will get back to you soonest.  

Mike – Please have staff look into the variations and determine which is correct.  Thanks

**Comment by Piglady64** [comment]
Here you go. Answer from Policy

Hi Julie,

We looked at USA Staffing the Feds Hire Vets wording and they are correct.  USAJOBS needs to follow the same and change your statement.

Thanks so much, have a great day!
-Michelle 

Michelle T. Glynn
Sr. Human Resources Specialist
U.S. Office of Personnel Management
Employee Services
Talent Acquisition and Workforce Shaping
Hiring Policy
202-606-1571

**Comment by danthibodeau** [comment]
I saw this was assigned to me, but not sure what's needed. I can put a ticket on the backlog, but do we have the desired language, or should it be exactly as it appears on Feds Hire Vets with the bullet points and bolding?

**Comment by danthibodeau** [comment]
As far as the PO taking this to improve, I don't know if anyone here understands the eligibility enough to do that. I'm eligible, and I have no clue what it means. Julie, since you're assigned, let me (or anyone else) know if you want any assistance. Likely though, we would reach back out to policy to help us with wording to confirm it's still accurate since none of us have the expertise.

**Comment by Piglady64** [comment]
I am going to set up a meeting with Feds Hire Vets, Policy and us to figure out what we need to say here.



Julie Jackson

Office of Personnel Management
USAJOBS PMO
Work : 202-606-5230
Cell Phone: 202-718-7608
Julie.jackson@opm.gov<mailto:Julie.jackson@opm.gov>

**Comment by Piglady64** [comment]
Questions for the meeting with Policy, Team Titan and Feds Hire Vets
(1) When they say Preference Eligible are they talking about the statute definition.
(2) If the question we are asking does not answer are they VEOA eligible what should the question be?

**Comment by dalenard21** [comment]
@danthibodeau @Piglady64 @humancompanion 

Here are notes from yesterdays meeting with Feds Hire Vets and Policy.  I'm sure I missed stuff - there was so much information.  Please feel free to add anything I missed.

Let's confirm next steps:

1. Debbie to try and rewrite the current veterans questions - @Piglady64 I may ask you to help me with this.
2. Review with Dan
3. Send to Policy and Feds Hire Vets (the people that were on the call) to review.
4. If all is good, Team Titan to open a ticket?  

@humancompanion will you want to prototype it out?  We may want to try that, since I think some of the questions will only appear if you answer them a certain way.

Thanks.
[Meeting-notes-Veteran-Preference-Policy_20180904.docx](https://github.com/USAJOBS/team-titan/files/2353971/Meeting-notes-Veteran-Preference-Policy_20180904.docx)

**Comment by Piglady64** [comment]
Questions for the meeting with Policy, Team Titan and Feds Hire Vets
(1) When they say Preference Eligible are they talking about the statute definition. 
      Yes this is what "Preference Eligible" is referring to
      It is a statute by Congress that "Preference Eligible" be included in the statement. (Katika)
      We need to mirror Feds Hire Vets - not the individual agencies
(2) If the question we are asking does not answer are they VEOA eligible what should the question be?
     We need to just add the "preference eligible" piece and then the statement is correct.

**Comment by humancompanion** [comment]
* "It is a statute by Congress that "Preference Eligible" be included in the statement." Fine. We'll abide by that but, that doesn't mean we'll fail to define what it means.
* "We need to just add the "preference eligible" piece and then the statement is correct." Yes, technically correct and totally unintelligible. You can't define eligibility by including the word eligible. That's ridiculous.

So we need a decision tree here to help users navigate the statute definition. @dalenard21 - Yes, we'll need to prototype it but, we could do that via Sketch and InVision. We could also show a decision tree diagram. I have an InDesign file in Google Drive > USAJOBS > Flow > USAJOBS.American_Discovery_Trail.profile_decision_tree.indd that captures earlier iterations on this, Veterans included. Start there.

**Comment by Piglady64** [comment]
Here is the link to DOL that Kelly Woodall from Feds Hire Vets sent me. I know we have looked at this in the past. Dan thought we used to link to this.  https://webapps.dol.gov/elaws/vets/vetpref/mservice.htm

**Comment by humancompanion** [comment]
If FedsHireVets can put us in touch with the folks responsible then we can port it over. Otherwise, we can duplicate it manually. But I don't believe USAJOBS should send folks off to Labor, or any other site, to answer questions on our site, with some exceptions. One exception might be if the VA included that "wizard" (for lack of a better term) and we had an API to them to import the answer. But, it's not so complicated that we couldn't reproduce it and I suspect we have far higher volume to our site than Labor.

**Comment by Piglady64** [comment]
I agree that we shouldn't send folks over to DoL. I think that is why we stopped it when we first started this re-design. One of our insights was something about people going to other sites to find information and one of our pillars (I think) was something about wanting to be the One Stop Shop for information about federal hiring.

**Comment by dalenard21** [comment]
@humancompanion @Piglady64 I somehow missed this communication, because I was heads down working on this.  So, yes, a decisionn tree would be good.  I sorta started on it in the attached word document. You'll see 'comments' where I put in when certain items would appear. I pulled out 'preference eligible" as it's own question and maybe we link to the statute.

I also used a cross between what's on the SF 15 and what's on FedsHireVets.  FedsHireVets gets more specific about the 'veterans' preference but I think the SF-15 does a pretty good  job of describing the family/derived preference.
[Profile-veterans-preference-questions-rewrite_20180906.docx](https://github.com/USAJOBS/team-titan/files/2358254/Profile-veterans-preference-questions-rewrite_20180906.docx)


Anyway, this is a start.  I can try to work on a decision tree if you think that's more helpful.

**Comment by dalenard21** [comment]
@humancompanion @Piglady64 here is a very rough decision tree, that breaks down the 'derived preference' piece of the flow.  These questions come from the SF 15 and are similar to what's asked in the advisor tool.

How does this impact the questions we ask?  Perhaps we don't ask if they have derived preference, but rather, we ask the specific questions and based on what they answer we tell them they may have derived preference.  This would make the veterans preference question page much longer. 

Other questions:

1. We currently ask them if they have Veteran's Preference.  Some may know immediately, whereas others may not, which is where we could link them to the advisor tool. And, at the end of the tool, we can say 'You may be eligible for "x" preference.  If we do this, do we tell the user to go back and select "yes" for veteran's preference and to select one of the choices we currently have?

2. Do we need to the question "Do you want to claim Veteran's Preference?"  Would someone say yes they have Veteran's preference but don't want to claim it?

3. Another option is to copy the advisor tool and at the end they repeat the answers and say whether someone may be eligible for a 5 point preference or 10 point preference, but they don't specify if it's 10% disability or over 30% disability.  Do we change our choices to just 5 point and 10 point?  If you look at the SF 15 it doesn't ask about 0 point preference (sole survivorship), so I'm not sure how we ask about that one?

Please review when you have a chance. I've also attached the SF 15.
[Veterans-preference-decision-tree_20180925.docx](https://github.com/USAJOBS/team-titan/files/2420340/Veterans-preference-decision-tree_20180925.docx)
[sf15.pdf](https://github.com/USAJOBS/team-titan/files/2420341/sf15.pdf)

**Comment by Piglady64** [comment]
@dlenard21 @humancompanion 
1. Yes we will need to tell them to go back and select the one that they feel applies. We can't make that choice for them in case they end up not being eligible for some reason. 

2. Yes we need to ask them if they want to claim veterans' preference. Some people don't want to claim it when they apply to certain jobs.

3. The 5 and 10 point preferences are merely hiring preferences. The 30% disability is a hiring authority so we do need to find out if they have a disability rating of 30% or more.
